review/fileToReview.js

```diff
@@ -11,36 +10,44 @@ var domain = "bank.local.fr"
  * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
  * @return {Object} All transactions available on the page
  */
-async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
-	console.log(`--- Fetch Trasactions page n°${page} ---`);
-	try {
-    var headers = {"Authorisation":  authorization }
```

Comment:
* change var for let because it form is changing later in the code
* spelling mistake 'AuthoriZation'